### ISSUE TITLE
anvio-master now requiring python package numpa?

### ISSUE THREAD

**Issue by TonyMane** [merenlab/anvio#1365]
```
Anvi'o version ...............................: esther (v6.1-master)
Profile DB version ...........................: 31
Contigs DB version ...........................: 14
Pan DB version ...............................: 13
Genome data storage version ..................: 6
Auxiliary data storage version ...............: 2
Structure DB version .........................: 1
```

Greetings, I have been using anvio (the master branch) for the past month or so with no issues. All the packages/scripts run as expected. Somewhat surprisingly, I tried to generate a contigs database earlier today, and found the following issue:

```
(anvio-master) (anvio-master) -bash-4.2$ anvi-gen-contigs-database -f pratice.fasta -o pratice.db
Traceback (most recent call last):
  File "/home/abertagnolli3/github/anvio/bin/anvi-gen-contigs-database", line 7, in <module>
    import anvio.dbops as dbops
  File "/home/abertagnolli3/github/anvio/anvio/dbops.py", line 27, in <module>
    import anvio.utils as utils
  File "/home/abertagnolli3/github/anvio/anvio/utils.py", line 27, in <module>
    from numba import jit
ModuleNotFoundError: No module named 'numba'
```

If found this strange. I also found the same error when running the master branch on another machine. I went back and looked through all the packages that anvio requires, i didn't see numba (but i checked pretty quick). Anyhow, i don't have admin rights on the machine i typically use to run anvio. It would appear this is required for numba. Any help would be useful.

Thank you!!!

**Comment by xvazquezc** [comment]
This was mentioned in the Slack channel. And yes, you need the numba Python package. `pip install numba==0.48.0` it is supposed to work. If you can't because you are not using a virtual environment, add the `--user` flag to that command. It will install the package in your user's directory. It won't be available to anyone else though.

**Comment by TonyMane** [comment]
Thank you! I clicked on the slack link (http://merenlab.org/2019/10/07/getting-help/#anvio-slack-non-technical) and added my email address. Haven't received email confirmation yet- can't seem to see anything on this channel yet. In the meantime, the numba installation worked, and the 'anvi-gen-contigs-database' command works. I ran 'anvi-run-hmms' as well. However, ran into another set of errors with the below:
(anvio-master) (anvio-master) -bash-4.2$ anvi-run-scg-taxonomy -c pratice.db 
Traceback (most recent call last):
  File "/home/abertagnolli3/github/anvio/bin/anvi-run-scg-taxonomy", line 58, in <module>
    groupH.add_argument(*anvio.A('write-buffer-size'), **anvio.K('write-buffer-size'))
  File "/home/abertagnolli3/github/anvio/anvio/__init__.py", line 2174, in A
    return D[param_id][0]
KeyError: 'write-buffer-size'
